### ISSUE TITLE
handling of goals assigned to central is wonky.

### ISSUE THREAD

**Issue by TarikViehmann** [carologistics/fawkes-robotino#644]
There is a somewhat hidden assumption the central agent that instruct goals are created with goal metas that have the "assigned-to" field set to "central".
This causes a bug in the current master (https://github.com/carologistics/fawkes-robotino/commit/2560adb3ee78f94de3802f220e37b35e4314b31a) due to a rule that reformulates one of these goals on failure: 
`goal-reasoner-evaluate-production-and-maintenance-wp-still-usable`
For goals assigned to robots the workflow is to unassign goals once they are finished, for goals assigned to central this is false.
This causes issues down the line as the goal will never be assigned to central again and thus gets stuck down the line (in the case that caused the creation of this bug, it was during goal expansion).

In summary,, we should decide on a coherent way of goal assignment and prevent issues like the one shown above.
One possibility would be to utilize the restricted-to field to manage instruct goals. 

Also, the specific issue with that rule should of course be fixed.
